nexus-repository-helm/src/main/java/org/sonatype/repository/helm/HelmFacet.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Sonatype Nexus (TM) Open Source Version
+ * Copyright (c) 2017-present Sonatype, Inc.
+ * All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+ *
+ * This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+ * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+ *
+ * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+package org.sonatype.repository.helm;
+
+import org.sonatype.nexus.repository.Facet;
+import org.sonatype.nexus.repository.storage.Asset;
+import org.sonatype.nexus.repository.storage.Bucket;
+import org.sonatype.nexus.repository.storage.Component;
+import org.sonatype.nexus.repository.storage.StorageTx;
+import org.sonatype.repository.helm.internal.HelmAssetAttributePopulator;
+import org.sonatype.repository.helm.internal.util.HelmAttributeParser;
+import org.sonatype.repository.helm.internal.util.HelmDataAccess;
+
+@Facet.Exposed
+public interface HelmFacet
+    extends Facet
+{
+  HelmDataAccess getHelmDataAccess();
```

Comment:
HelmFacet now seems to be like proxy for these 3 helper objects. But it's not how this class supposed to be used. It violates IoC rules and seems to be redundant. It's better to `@Inject` these objects where they are needed directly.

Reply:
Done